backend/api_v2/serializers.py

```
@@ -248,6 +250,20 @@ def validate_hitl_queue_name(self, value: str | None) -> str | None:
+        """Validate packet ID format using enterprise validation if available."""
+        if not value:
+            return value
+
+        # Check if HITL feature is available using FeatureRegistry
+        if not FeatureRegistry.is_hitl_available():
+            raise ValidationError(
+                "Packet-based HITL processing requires Unstract Enterprise. "
+                "This advanced workflow feature is available in our enterprise version. "
+                "Learn more at https://docs.unstract.com/unstract/unstract_platform/features/workflows/hqr_deployment_workflows/ or "
+                "contact our sales team at https://unstract.com/contact/"
+            )
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)